react/src/components/MainLayout/WebUIHeader.tsx

```diff
@@ -111,20 +118,35 @@ const WebUIHeader: React.FC<WebUIHeaderProps> = ({
             }}
           />
         </Suspense>
-        <Flex direction="row" className="non-draggable">
-          <BAINotificationButton />
+      </Flex>
+      <Flex direction="row" className="non-draggable" gap="xxs" align="center">
+        {baiClient.supports('extend-login-session') &&
+          baiClient._config.enableExtendLoginSession && (
+            <Suspense>
+              <LoginSessionExtendButton />
+              {gridBreakpoint.md && (
+                <Divider
+                  type="vertical"
+                  style={{ borderColor: 'transparent' }}
+                />
+              )}
+            </Suspense>
+          )}
+        <BAINotificationButton
+          buttonRender={(btn) => (
+            <ReverseThemeProvider>{btn}</ReverseThemeProvider>
+          )}
+        />
```

Comment:
Is there a reason you only added bottenRender to certain buttons?

Reply:
@ironAiken2 In the NEO UI, the header uses a reversed theme: white text in light mode and black text in dark mode. Therefore, we need to apply the reversed theme to the button only, excluding the Dropdown and Notification drawer.
With the `buttonRender`, we can wrap the button depending on the situation.

Reply:
Is this different from wrapping it in a ReserveThemeProvider without a buttonRender?
`
<ReserveThemeProvider>
     <BAINotificationButton />
</ReserveThemeProvider
`